src/app/news/components/post.tsx

```diff
@@ -30,31 +31,30 @@ export function Post({
             </a>
           </p>
           <p
-            className="flex items-center gap-1 text-sm text-gray-500"
-            title={`last updated: ${new Date(
+            className="flex items-center gap-1 text-xs text-gray-500 md:text-sm"
+            title={`ostatnia modyfikacja: ${new Date(
               post.updated_time,
             ).toLocaleString()}`}
           >
             {new Date(post.created_time).toLocaleString()}
```

Comment:
można użyć polskiej daty ewentualnie tu
```suggestion
           {new Date(post.created_time).toLocaleString("pl-PL", {
              day: "numeric",
              month: "long",
              hour: "2-digit",
              minute: "2-digit",
            })}
```